lib/src/components/ComponentWrapper.tsx

```diff
@@ -15,11 +22,11 @@ export class ComponentWrapper {
         this._saveComponentRef = this._saveComponentRef.bind(this);
         this.state = {
           componentId: props.componentId,
-          allProps: _.merge({}, props, store.getPropsForId(props.componentId))
```

Comment:
I believe we need `props` and `id` here as well, but I guess a unit test will need to prove it..

Reply:
👍